concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -345,25 +336,104 @@ jobs:
           params:
             file: updated-bucket-state/bucket.tfstate
 
-      - task: self-update-pipelines
-        tags: [colocated-with-web]
+      - try:
+          on_error:
+            task: check-if-bootstrap-fail-if-not
+            config:
+              platform: linux
+              image_resource: *gov-paas-bosh-cli-v2-image-resource
+              outputs:
+                - name: check-if-bootstrap
+              run:
+                path: sh
+                args:
+                  - -e
+                  - -c
+                  - |
+                    echo "${TARGET_CONCOURSE}"
+
+                    if [ "$TARGET_CONCOURSE" = "bootstrap" ]; then
+                      echo This concourse is bootstrap, passing this task
+                      echo pass > check-if-bootstrap/concourse.txt
```

Comment:
I think the `check-if-bootstrap` name isn't descriptive enough of what this is being used for. It's actually more like "ignore-self-update-pipelines-failure" or something.

Reply:
41a5fbe6a81976514900ca70ba583276a55c3219